From 78b20f8ff0a29492040ee0a3fd03cbb11568dcdc Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 1 Jul 2019 10:46:01 +0800
Subject: [PATCH] Mount /var/volatile during install

The installed system needs /var/volatile clean, otherwise it
caused systemd's journalctl failed to record boot log.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/modules/storage/devicetree/fsset.py | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/pyanaconda/modules/storage/devicetree/fsset.py b/pyanaconda/modules/storage/devicetree/fsset.py
index dc4dab34f..ba66f529e 100644
--- a/pyanaconda/modules/storage/devicetree/fsset.py
+++ b/pyanaconda/modules/storage/devicetree/fsset.py
@@ -131,6 +131,13 @@ def get_system_filesystems(devicetree):
                 mountpoint="/sys/fs/selinux"
             )
         ),
+        NoDevice(
+            fmt=get_format(
+                "tmpfs",
+                device="tmpfs",
+                mountpoint="/var/volatile"
+            )
+        ),
         NoDevice(
             fmt=get_format(
                 "usbfs",
-- 
2.7.4

